src/utils/saml.ts

```diff
@@ -326,7 +332,7 @@ export const getSamlOptions: MultiSamlConfig["getSamlOptions"] = (
     }
     const authOptions = maybeAuthOptions.getOrElse({});
     const options = {
-      ...spidStrategyOptions.sp,
+      ...maybeSpidStrategyOptions.value,
```

Comment:
why sp -> value ?

Reply:
You are right. It's an error. Must be `maybeSpidStrategyOptions.value.sp`

Reply:
Fixed in 031a6f1